Sources/ParseLiveQuery/Internal/ClientPrivate.swift

```diff
@@ -228,7 +227,7 @@ extension Client {
             switch response {
             case .Connected:
                 self.subscriptions.forEach {
-                    self.sendOperationAsync(.Subscribe(requestId: $0.requestId, query: $0.query))
+                    self.sendOperationAsync(.Subscribe(requestId: $0.requestId, query: $0.query, sessionToken: PFUser.currentUser()?.sessionToken))
```

Comment:
Since this is a loop, there's the weird off chance that another thread changes the current user between iterations of the loop - let's cache `sessionToken` outside the loop, so all of our queries get subscribed to the same user (should also improve perf. slightly).